README.md

```diff
@@ -1 +1,68 @@
-# Microverse-Gaming-Convention
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Kindly try to fix the overlapping problem in the mobile version when I press the more button one of the speakers overlaps the brand's section.
![image](https://user-images.githubusercontent.com/24703171/163245165-2aefbece-7492-4726-9254-a568ad0837b7.png)